src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -2320,7 +2321,15 @@ export function getChartSettingsMap(
             chartSettingsMap[
                 attribute.uniqueKey
             ].filterByCancerGenes = filterCNAGenesTableByCancerGenes;
+        } else if (chartType === ChartTypeEnum.BAR_CHART) {
+            debugger;
+            if (getNAValue) {
+                chartSettingsMap[attribute.uniqueKey].showNA = getNAValue(
+                    attribute.uniqueKey
+                )();
```

Comment:
is `()` needed here?

Reply:
`()` is used to call the lambda returned by the `getNAValue` call, which returns a boolean to assign to `showNA`

Reply:
I found out this caused a cycle, so I revised this code in the next commit